cli/src/client/mod.rs

```diff
@@ -43,13 +34,29 @@ impl Client {
             .header("apollographql-client-name", "experimental-apollo-cli")
             .header(
                 "apollographql-client-version",
-                env::var("CARGO_PKG_VERSION").unwrap(),
+                env::var("CARGO_PKG_VERSION").unwrap_or("testing".into()),
```

Comment:
Could we have a better default than `"testing"` ?

Reply:
Switched it to `dev`